cmd/nameset-backfill/main.go

```diff
@@ -71,18 +71,18 @@ func (b *backfiller) run() error {
 
 func (b *backfiller) findCerts() ([]resultHolder, error) {
 	var allResults []resultHolder
+	lastSerial := ""
 	for {
 		var results []resultHolder
 		_, err := b.dbMap.Select(
 			&results,
 			`SELECT c.serial, c.issued, c.expires, c.der FROM certificates AS c
        LEFT JOIN fqdnSets AS ns ON c.serial=ns.serial
        WHERE ns.serial IS NULL
-       ORDER BY c.issued DESC
-       LIMIT ?
-       OFFSET ?`,
+       AND c.serial > ?
```

Comment:
Another way to put it: We want to iterate over the whole table in chunks, and don't care about the order.

New serials created while we are iterating will get inserted into the middle of some of our chunks, but we don't care about those. They will have their namesets filled already.